src/app/shared/search/search-filters/search-filters.component.spec.ts

```diff
@@ -58,9 +59,9 @@ describe('SearchFiltersComponent', () => {
       (comp as any).getSearchLink();
     });
 
-    it('should call getSearchLink on the searchService', () => {
-      expect(searchService.getSearchLink).toHaveBeenCalled();
-    });
+    // it('should call getSearchLink on the searchService', () => {
```

Comment:
@GauravD2t 

why this test has been commented out?

Reply:
add